### DISCUSSION TITLE
Github Actions self hosted runner not working on RHEL8

### DISCUSSION THREAD

**Discussion by gautamkrishnar** [community/community#27079] [Actions]
<p>Github Actions self hosted runner is not working on RHEL 8 It is stuck while running the config script on the following step:</p>
<pre><code class="lang-auto">$ ./config.sh --url &lt;repo&gt; --token &lt;Token&gt;

--------------------------------------------------------------------------------
|        ____ _ _   _   _       _          _        _   _                      |
|       / ___(_) |_| | | |_   _| |__      / \   ___| |_(_) ___  _ __  ___      |
|      | |  _| | __| |_| | | | | '_ \    / _ \ / __| __| |/ _ \| '_ \/ __|     |
|      | |_| | | |_|  _  | |_| | |_) |  / ___ \ (__| |_| | (_) | | | \__ \     |
|       \____|_|\__|_| |_|\__,_|_.__/  /_/   \_\___|\__|_|\___/|_| |_|___/     |
|                                                                              |
|                       Self-hosted runner registration                        |
|                                                                              |
--------------------------------------------------------------------------------

# Authentication
</code></pre>
<p>Anyone else faced this problem. I cant find a verbose mode to debug this issue on config script.</p>

**Comment by joshmgross** [comment, marked as answer]
<p>:wave: Hey <a class="mention" href="/gautamkrishnar">@gautamkrishnar</a>,</p>
<p>Looks like your question was answered in <a href="https://github.com/actions/runner/issues/997" class="inline-onebox">Github Actions self hosted runner not working on RHEL 8 · Issue #997 · actions/runner · GitHub</a> :tada:</p>
<p>For anyone else looking at this issue, see <a href="https://github.com/actions/runner/issues/997#issuecomment-848899281" class="inline-onebox">Github Actions self hosted runner not working on RHEL 8 · Issue #997 · actions/runner · GitHub</a></p>
